static/js/components/widgets/ResourcePickerDialog.tsx

```diff
@@ -29,12 +30,23 @@ interface Props {
 
 interface TabSettings {
   title: string
-  id: TabIds
-  contentType: ContentType
+  id: string
+  contentType: string
   resourcetype: string | null
   embeddable: boolean
   singleColumn: boolean
-  sourceWebsiteName?: string
+}
+
+export function toTabSettings(elt: string[]) {
```

Comment:
The argument `elt` should always have exactly two elements, right? 

**Requested Change:** If yes, let's just use two arguments with meaningful names, e.g., `toTabSettings(contentType, title)`.

Reply:
Updated in https://github.com/mitodl/ocw-studio/pull/1448/commits/b46ad80309a35e46f7487f9c547179a8bcbe6fe2.